hyp3_testing/helpers.py

```diff
@@ -0,0 +1,91 @@
+import os
+import json
+import random
+import string
+from glob import glob
+from pathlib import Path
+from zipfile import ZipFile
+
+import requests
+from hyp3lib.fetch import download_file
+from jinja2 import Template
+
+from hyp3_testing import AUTH_URL
+
+
+def hyp3_session(username: str = None, password: str = None):
+    if username is None:
+        username = os.environ.get('HYP3_USERNAME')
+    if password is None:
+        password = os.environ.get('HYP3_PASSWORD')
+
+    session = requests.Session()
+    resp = session.get(AUTH_URL, auth=(username, password))
+    resp.raise_for_status()
+
+    return session
+
+
+def get_submission_payload(template):
+    with open(template) as f:
+        body = f.read()
+
+    name = ''.join(random.choices(string.ascii_letters + string.digits, k=7))
+    jobs = Template(body).render(name=name)
+
+    return json.loads(jobs)
+
+
+def get_jobs_update(name, url, session, request_time=None):
+    params = {'name': name}
+    if request_time is not None:
+        params.update({'start': request_time, 'end': request_time})
```

Comment:
do we need to filter by time if we're already filtering based on an auto-generated name?

Reply:
We probably don't need to, but we could have submissions name collisions